.github/workflows/ci.yml

```diff
@@ -59,7 +59,7 @@ jobs:
       - uses: Swatinem/rust-cache@v2
 
       - name: Setup postgres for tests
-        run: docker-compose up -d
+        run: docker compose up -d
```

Comment:
apparently `docker-compose` is deprecated and you're supposed to use `docker compose` now. tests were failing without this change